src/Octokit.js

```diff
@@ -0,0 +1,35 @@
+// MIT © 2021 vbouchet31
+"use strict";
+const { Octokit } = require("@octokit/rest");
+
+export default class OctokitClient {
+    constructor(token) {
+        this.octokit = new Octokit({
+            auth: token,
+            timeZone: "Europe/Amsterdam",
```

Comment:
Probably,  it is good that `timeZone` is the default(no specify)
https://docs.github.com/en/rest/overview/resources-in-the-rest-api#timezones